packages/openapi-v3/test/unit/to-openapi-path.unit.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3.
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+
+import {toOpenApiPath} from '../../src/controller-spec';
+
+describe('toOpenApiPath', () => {
+  it('allows /:foo/bar', () => {
+    const path = toOpenApiPath('/:foo');
+    expect(path).to.eql('/{foo}');
+  });
+
+  it('allows /:foo/bar', () => {
```

Comment:
We're going to start supporting express style of parameters? If so, we should document it.

Reply:
We didn't do any check before this PR. I think it will be good to allow express style too.

Reply:
> We didn't do any check before this PR. I think it will be good to allow express style too.

I would rather not introduce multiple parameter styles, I am concerned about the confusion it can create and the extra work it will add for any consumer of routes.